CMakeLists.txt

```diff
@@ -2101,7 +2101,7 @@ elseif(APPLE)
         #import <Metal/Metal.h>
         #import <QuartzCore/CAMetalLayer.h>
 
-        #if (!TARGET_CPU_X86_64 && !TARGET_CPU_ARM64)
+        #if (TARGET_OS_OSX && !TARGET_CPU_X86_64 && !TARGET_CPU_ARM64)
```

Comment:
Does Metal work on 32 bit iOS apps? I thought it didn't, but maybe I'm just confusing that with macOS.

Reply:
it doesn't as well, but, like I wrote in PR description, Metal feature will be disabled completely (even for arm64) if one builds SDL with `-D CMAKE_OSX_ARCHITECTURES="armv7;arm64"`. Metal for 32-bit iOS is disabled in `SDL_config_iphoneos.h` anyway.

Reply:
I'm not too familiar with that cmake build setup (I typically use `lipo`). Will that generate a SDL_config which enables Metal on both armv7 and arm64? I thought the hand-written `SDL_config_iphoneos.h` wasn't used when cmake is used.

Reply:
> I thought the hand-written `SDL_config_iphoneos.h` wasn't used when cmake is used.

You're absolutely right, thanks for bringing my attention to this! I didn't check the CMake file deeply.

> thus potentially causing issues on armv7 where it doesn't work, apparently

I've checked SDL code shortly: yes, it might be possible.